src/test/scala/uk/gov/nationalarchives/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -85,6 +90,94 @@ class ExternalServicesTestUtils extends AnyFlatSpec with BeforeAndAfterEach with
   val defaultEntitiesWithSourceIdReturnValues: List[IO[Seq[Entity]]] =
     List(IO(structuralObjects(0)), IO(structuralObjects(1)), IO(structuralObjects(2)))
 
+  val missingTitleInDbScenarios
+      : TableFor6[String, Option[String], Option[String], Option[String], Option[String], String] = Table(
+    (
```

Comment:
There's no check here for a `None` title in Preservica. Can this happen?

Reply:
I don't think so